curvefs/src/mds/schedule/copysetSchedule.cpp

```diff
@@ -222,8 +222,62 @@ int CopySetScheduler::CheckAndBalanceZone(ZoneIdType zoneId) {
             if (!IsCopysetCanTransfer(copyset)) {
                 continue;
             }
-            LOG(INFO) << "transfer copyset from metaserver " << sourceId
-                      << " to metaserver " << destId;
+            LOG(INFO) << "transfer copyset from high usage metaserver "
+                      << sourceId << " to low usage metaserver " << destId;
+            if (TransferCopyset(copyset, sourceId, destId)) {
```

Comment:
Does here try to transfer all copysets on high usage metaserver to lower one with a for loop? If so, the lower one becomes higher.

Reply:
```
            if (TransferCopyset(copyset, sourceId, destId)) {
                genOpCount++;
                return genOpCount;
            }
```
If tranfer copyset gen op success, it will return. So only gen one op here.